### DISCUSSION TITLE
sound problems in chromium-based internal browser

### DISCUSSION THREAD

**Discussion by python2209** [obsproject/obs-studio#9030] [Development Questions]
the short alert sounds used for streaming do not play in the internal browser of obs chromium version. The old obs studio browser in 26.0.0 worked.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
